### DISCUSSION TITLE
No blur

### DISCUSSION THREAD

**Discussion by ALBINPRAVEEN** [tsujan/Kvantum#772] [Q&A] [answered]
When i apply a theme it is not applying blur or optacity also aps are not getting transparent. Im new to linux amd kde, can u pls help me out sir..

**Comment by tsujan** [comment, marked as answer]
Your question is too general.

Check if you have transparency/blurring anywhere. If not, check your graphic settings. If you have transparency/blurring elsewhere, check whether the Kvantum theme you use has translucency/blurring enabled.

**Reply by ALBINPRAVEEN** [reply]
Enabled but then also not working

**Comment by bluee111** [comment]
Hello, Here same problem with the Blur, it seems that it only happens with the last kde plasma 5.24

**Reply by ALBINPRAVEEN** [reply]
Bro i have found the fix solution it is not prblm with the kvantum

System Settings → Hardware → Display and Monitor. 
For the best results, you should choose OpenGL 3.1. Also note that the blur effect does not work with the XRender setting.

**Reply by bluee111** [reply]
thanks, that makes sense. but it is strange that now it is not available.. hmm but this is already a problem on my pc, thanks for that quick response. https://i.ibb.co/7WYq61C/2022-04-01-20-31.png

**Reply by bluee111** [reply]
Fixed, I was missing to enable "Blurring for translucent windows" from Kavantum manager > Compositing & General Look
Sorry for the inconvenience :D
